src/pudl/transform/eia860.py

```diff
@@ -582,8 +582,8 @@ def transform(eia860_raw_dfs, eia860_tables=pc.pudl_tables["eia860"]):
     # for each of the tables, run the respective transform funtction
     for table in eia860_transform_functions:
         if table in eia860_tables:
-            logger.info(f"Transforming raw EIA 860 DataFrames for {table} "
-                        f"concatenated across all years.")
+            logger.info("Transforming raw EIA 860 DataFrames for %s "
```

Comment:
Out of curiosity, is there something wrong with f-strings here?

Reply:
My understanding is that f-strings always get interpolated, whereas giving the variables as arguments to the logger method prevents them from getting interpolated if you're not at that log level. And this can be important if you're only assigning to those variables based on the logging level (e.g. you don't bother calculating value unless you're at DEBUG) If you search for Python lazy logging I think you'll find more info https://stackoverflow.com/questions/13500813/how-to-use-modern-string-formatting-options-with-pythons-logging-module

Reply:
Huh, I didn't know that. Following your link, somebody [made the argument](https://github.com/PyCQA/pylint/issues/2354) that because f-strings are significantly faster to interpolate, the savings from avoided interpolation of suppressed log levels are partly offset by slower interpolation at higher levels, plus the cost of reduced readability. And that the absolute time is very small difference either way unless you're churning out millions of logs.

I take your point that it is a neat trick to keep in mind if `table` was an expensive calculation only made in the logging call.

I guess my takeaway is that it isn't a hill I'm willing to die on so either way is fine with me 😆

Reply:
Truth be told, I was just following the orders given to me by the linters! Personally I prefer using f-strings as I find them much more readable. So if there's no clear "right way" to do this then I'd love to use f-strings across the board and turn off that linter error.  The one place where I could imagine an actual error coming up is when some variable is only defined *contingent upon* a certain loglevel being set (e.g. don't generate a JSON report to print out unless you're at DEBUG) in which case attempting to interpolate the unset variable would cause a problem. But I don't think we're actually doing this anywhere, are we?